### PR TITLE
User access CRUD changed for user 

### PR DESCRIPTION
**Pull request by naumvd95**
- CRUD configuration Docs updated

### CODE REVIEW

File: docs/kqueen.rst [hunk 1]
```diff
@@ -159,6 +159,41 @@ pattern.
...
+- Login.
+- ``CRUD`` all Organizations.
+- ``CRUD/Manage`` all members.
+- ``CRUD/Manage`` all member roles.
+- Full-Admin rights.
+
+Admin view
+~~~~~~~~~~
+
+- Login.
```
Comment (ekhomyakova) [inline docs/kqueen.rst]: This is included into "Full-User rights"

File: docs/kqueen.rst [hunk 2]
```diff
@@ -159,6 +159,41 @@ pattern.
...
+
+Default User Access configuration
+_________________________________
+
+Default CRUD (Create, Read, Update, Delete) model for Kqueen user roles.
+
+Superadmin view
+~~~~~~~~~~~~~~~
+
+- Login.
```
Comment (ekhomyakova) [inline docs/kqueen.rst]: This is included into "Full-User rights"

File: docs/kqueen.rst [hunk 3]
```diff
@@ -159,6 +159,39 @@ pattern.
...
+- Full-User rights.
+
+
+User view
+~~~~~~~~~
+
+- Login.
+- ``R`` Organization members.
+- ``R`` Provisioners.
+- ``CRUD`` self Clusters.
```
Comment (katyafervent) [inline docs/kqueen.rst]: Currently user is able to manage provisioners too, should we change code first if we decided that it is a correct behavior. Also I've just remember, that policy can be set up for a particular organization,  not sure that it is tested, but now organization has policy settings

Comment (katyafervent) [reply docs/kqueen.rst]: I suggest to add CRUD provisiners here, and  change docs together with user right restriction

Comment (naumvd95) [reply docs/kqueen.rst]: IMO user can't manage provisioners at all
need to deny provisioner settings for user

File: docs/kqueen.rst [hunk 4]
```diff
@@ -159,6 +159,39 @@ pattern.
       - 127.0.0.0/8
       - Addresses allowed to access metrics endpoint without token
 
+
+Default User Access configuration
```
Comment (mzlatkova) [inline docs/kqueen.rst]: Default user access configuration

File: docs/kqueen.rst [hunk 5]
```diff
@@ -159,6 +159,39 @@ pattern.
...
+_________________________________
+
+Default CRUD (Create, Read, Update, Delete) model for Kqueen user roles.
```
Comment (mzlatkova) [inline docs/kqueen.rst]: KQueen

File: docs/kqueen.rst [hunk 6]
```diff
@@ -159,6 +159,39 @@ pattern.
...
+
+Superadmin view
+~~~~~~~~~~~~~~~
+
+- ``CRUD`` all Organizations.
```
Comment (mzlatkova) [inline docs/kqueen.rst]: organizations.

File: docs/kqueen.rst [hunk 7]
```diff
@@ -159,6 +159,39 @@ pattern.
...
+- ``CRUD/Manage`` all members.
+- ``CRUD/Manage`` all member roles.
+- Full-Admin rights.
```
Comment (mzlatkova) [inline docs/kqueen.rst]: Full admin rights.

File: docs/kqueen.rst [hunk 8]
```diff
@@ -159,6 +159,39 @@ pattern.
...
+
+Admin view
+~~~~~~~~~~
+
+- Invite/Remove members in own Organization (Email/LDAP).
```
Comment (mzlatkova) [inline docs/kqueen.rst]: Invite/remove members in own organization

File: docs/kqueen.rst (hunk 3 above)
Comment (katyafervent) [reply docs/kqueen.rst]: ok, so let's make it first and then push the docs

File: docs/kqueen.rst [hunk 9]
```diff
@@ -159,6 +159,39 @@ pattern.
...
+- ``CRD`` all Provisioners.
```
Comment (mzlatkova) [inline docs/kqueen.rst]: provisioners (no caps) please check below as well

File: docs/kqueen.rst (hunk 3 above)
Comment (naumvd95) [reply docs/kqueen.rst]: okok, its better to deny from UI part?

Comment (katyafervent) [reply docs/kqueen.rst]: I believe it is set up in policy json in config (in api) section and should be real quick

**Comment by naumvd95** [comment]
@katyafervent done

**Pull request merged**